app-misc/pet/pet-0.3.0.ebuild

```diff
@@ -33,7 +35,8 @@ KEYWORDS="~amd64"
 IUSE="zsh-completion"
 
 # dev-go/toml doesn't provide sources
-DEPEND="dev-go/go-crypto:=
+DEPEND="dev-lang/go:=
```

Comment:
Doesn't golang-build handle depending on go?

Reply:
Yes, it inherits golang-base, which has a dependency on go.
dev-lang/go removed.